.travis.yml

```diff
@@ -3,22 +3,25 @@ sudo: false
 language: c
 
 env:
-  - GHCVER=7.8.4
-  - GHCVER=7.10.2
+  - GHCVER=7.8.4 CABALVER=1.22
```

Comment:
It works. I can change it to use `1.24`.

Actually then one could change to use [`new-build`](http://blog.ezyang.com/2016/05/announcing-cabal-new-build-nix-style-local-builds/) which would make `tinc` obsolete. But maybe not too much changes at once